pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -319,8 +319,14 @@ func handleEvent(s *Server) {
 
 	// Only updating intermediate CA is supported now
 	if !bytes.Equal(currentCABundle, newCABundle) {
-		log.Info("Updating new ROOT-CA not supported")
-		return
+		// Check if newCABundle or currentCABundle is combination of other
```

Comment:
please be more clear:  in order to support root ca rotate, we need to make the new CA bundle contain both old and new CA certs